habitat-baselines/habitat_baselines/config/habitat_baselines/rl/agent/self_play.yaml

```diff
@@ -0,0 +1,4 @@
+name: "MultiAgentAccessMgr"
```

Comment:
I think agent as the name of the folder is going to be confusing. I would stick to agent_manager or multi_agent_setup instead. (agent is already used in habitat-lab config)

Reply:
This is out of scope for this PR. This change was made in the Part 1 PR which was targeting main. I can create a PR for main with this fix.